### DISCUSSION TITLE
save cookies to disk for reuse

### DISCUSSION THREAD

**Discussion by MikhailKlemin** [go-rod/rod#309] [General]
Hi there!

If there a way or workaround to save cookies for later load up? 

There is `browser.MustGetCookies()` which returns slice of pointers which is not easily serializable I think? 

May be switching profiles, but that also not quite clear how to do, and would not be overkill  for the task.

**Comment by ysmood** [comment]
I think `json.Marshal` and `json.Unmarshal` will do the job.

**Reply by MikhailKlemin** [reply]
Yes I just  tried and it looks so! 
I do not know why I though there is a problem with pointers... 
May be some earlier Go version had an issue with that, or I just dreemed it on
